modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/hair.dm

```diff
@@ -395,6 +395,86 @@
 	name = "Long Over Eye Alt"
 	icon_state = "hair_longovereye_alt"
 
+/datum/sprite_accessory/hair/skyrat/gentle_bun
+	name = "Gentle Bun"
+	icon_state = "hair_gentle_bun"
+
+/datum/sprite_accessory/hair/skyrat/stacy
+	name = "Stacy"
+	icon_state = "hair_stacy"
+
+/datum/sprite_accessory/hair/skyrat/stacy_bun
+	name = "Stacy Bun"
+	icon_state = "hair_stacy_bun"
+
+/datum/sprite_accessory/hair/skyrat/braided_front
+	name = "Braided Front"
+	icon_state = "hair_braided_front"
+
+/datum/sprite_accessory/hair/skyrat/phoenix
+	name = "Phoenix"
+	icon_state = "hair_phoenix"
+
+/datum/sprite_accessory/hair/skyrat/spikey
+	name = "Spikey"
```

Comment:
```suggestion
	name = "Spikey"
```
One letter off from an upstream hair. And unless I'm blind I can't see a hair_spikey in your DMI.

Reply:
Ignore the "can't see a hair_spikey in your DMI.". Saw it.

Still very close to an existing hair. Needs a more distinct name.